identity/credentials_code.go

```diff
@@ -7,7 +7,7 @@ import (
 	"database/sql"
 )
 
-type CodeAddressType string
+type CodeAddressType = string
```

Comment:
This is now a type alias - is that what we want here?

Reply:
Yes, because these types don't really make sense anymore, with the change to the "channel" based approach. The longer term goal is to make this a string entirely and configure how to get the code to the user using the `"via"` argument in the identity schema.